streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -489,7 +490,11 @@ private void commitAll() {
     private void commitOne(AbstractTask task, long now) {
         try {
             task.commit();
+        } catch (CommitFailedException e) {
```

Comment:
That is true, this is the only place throwing this exception. I feel the exception message "... due to group rebalance" is a bit narrow. BTW can we change the comment to "commit failed due to incorrect membership. Just log it"? Otherwise it is not very self-distinguishing with the other comment (line 497).